i18n-helpers/src/directives.rs

```diff
@@ -0,0 +1,130 @@
+use regex::{escape, Regex};
+
+#[derive(Debug, PartialEq)]
+pub enum Directive {
+    Skip,
+    TranslatorComment(String),
+}
+
+pub struct DirectiveFinder {
```

Comment:
With my idea below of hard-coding the regex, you might not need this struct. You can add methods to enums just like with strucst, so I suggest adding a

```rust
pub fn find(html: &str) -> Option<Self>
```

method to `Directive`.

Reply:
Acknowledged.  Demoted the method to a plain function in the `directives` module.